core/storage/certificate_assessment/gae_models.py

```
@@ -274,3 +276,265 @@ def create(
+            'The id generator for CertificateAssessmentResponseModel '
+            'is producing too many collisions.'
+        )
+
+    @classmethod
+    def create(
+        cls,
+        attempt_id: str,
+        question_id: str,
+        question_version: int,
+        selected_answer: str,
+        is_correct: bool,
```

Ruff (0.15.20)

[warning] Boolean-typed positional argument in function definition

(FBT001)